lighthouse/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub use account_manager;
+pub use beacon_node;
+pub use boot_node;
+pub use database_manager;
+pub use environment;
+pub use eth2_network_config;
+pub use lighthouse_network;
+pub use lighthouse_version;
+pub use slasher;
+pub use task_executor;
+pub use types as eth2_types;
+pub use validator_client;
+pub use validator_manager;
```

Comment:
IMO other useful additions would be:

- `eth2` crate (maybe renamed to `http`, it's our HTTP client for the beacon-API)
- `store` crate instead of `database_manager`. The DB manager is just the CLI subcommand, `store` is where all the good stuff happens.
- `slot_clock` for time management
- `execution_layer` for EL interop types & APIs

These are based on the things I commonly use externally, e.g. in `eleel`:

https://github.com/sigp/eleel/blob/cfc8ea2fe73d6a1835b6eb7625685635d01ff37e/Cargo.toml#L7-L12

And in `blockdreamer`:

https://github.com/blockprint-collective/blockdreamer/blob/030475d95cc46cab8c36b776bfa1738217b1ba25/Cargo.toml#L23-L27

Reply:
Addressed in [`1a4e548`](https://github.com/sigp/lighthouse/pull/5875/commits/1a4e548090d355443eede81f55c8f8502ccfbbf6).